core/templates/services/contextual/url.service.spec.ts

```diff
@@ -22,11 +22,13 @@ import { UrlService } from 'services/contextual/url.service';
 import { WindowRef } from './window-ref.service';
 
 describe('Url Service', () => {
-  let urlService: UrlService = null;
-  let windowRef: WindowRef = null;
+  let urlService: UrlService;
+  let windowRef: WindowRef;
   let sampleHash = 'sampleHash';
   let pathname = '/embed';
-  let mockLocation = null;
+  // Check https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys.
```

Comment:
Can we explain a bit about `Pick` in a single or two line here?

Reply:
I do think it is pretty self explanatory and not needed. We were not explaining `Record<>` anywhere in the code either, since it is a built-in type for typescript.